src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  display: block;
+  background-color: #fff;
+  border-radius: 5px;
+}
+
+.image {
+  width: 160px;
+  height: 134px;
+  background-image: url(/src/images/imac.jpeg);
+  background-size: 160px 134px;
+  margin-top: 32px;
+  margin-left: 19px;
+  margin-right: 19px;
+}
+
+.title {
+  width: $items-width;
+  height: 36px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+  color: #060b35;
+  margin-top: 40px;
+  margin-left: $items-left-margin;
+}
+
+.product_code {
+  margin-top: 4px;
+  margin-left: $items-left-margin;
+  width: 98px;
+  height: 14px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.button {
+  width: 166px;
+  height: 40px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+  background-color: #00acdc;
+  color: #fff;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  margin-left: 16px;
+  margin-top: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.price {
+  width: $items-width;
+  display: flex;
+  justify-content: space-between;
+  margin-left: $items-left-margin;
```

Comment:
you need move some values to variables for two reasons. First, it repeat. Second, if you need describe value for best readability ( for example magic numbers or something which will must be described).  

In this case, I think, you can left only value.